suite-common/wallet-core/src/transactions/transactionsReducer.ts

```diff
@@ -200,16 +202,37 @@ export const selectTransactionFirstTargetAddress = memoizeWithArgs(
     },
 );
 
-export const selectTransactionFirstInputAddress = memoizeWithArgs(
+const selectTransactionTargetAddresses = memoizeWithArgs(
```

Comment:
Please add `size` args for all these selectors.

Reply:
size args applied in 8e7e8f721